### ISSUE TITLE
Google voices stops speaking after a few words

### ISSUE THREAD

**Issue by sreejith-pm** [skechboy/SpeakIt#26]
The google voices given in "SpeakIt" extension of google chrome are not working correctly. It starts reading but stops after a few words. I've tried Google Voice UK English Male and Female.
Please correct this issue ASAP.

**Comment by Jasbick** [comment]
Same problem here. I've tried "Google português do Brasil".

**Comment by JamesNewton** [comment]
I don't think this can be corrected by the author. Google voices are not licensed to be used for these applications, and google is probably the source of the issue as they protect their network resources from "abuse". Use the native or SpeakIt TTS engines.
